inc/namespace.php

```diff
@@ -196,3 +198,77 @@ function enqueue_scripts() {
 	// Print queued scripts.
 	print_head_scripts();
 }
+
+/**
+ * Schedule index maintenance daily.
+ */
+if( ! wp_next_scheduled( __NAMESPACE__ . '\\index_maintenance' ) ) {
+	wp_schedule_event( time(), 'daily',  __NAMESPACE__ . '\\index_maintenance' );
```

Comment:
Do think it'd make sense to get the nearest time at midnight? So indexes are only removed at the end of a day rather than whatever time this code runs initially?